stdlib/LinearAlgebra/src/generic.jl

```diff
@@ -1516,22 +1512,11 @@ end
         throw(DimensionMismatch("reflector has length $(length(x)), which must match the first dimension of matrix A, $m"))
     end
     m == 0 && return A
-    @inbounds begin
-        for j = 1:n
-            # dot
-            vAj = A[1, j]
-            for i = 2:m
-                vAj += x[i]'*A[i, j]
-            end
-
-            vAj = conj(τ)*vAj
-
-            # ger
-            A[1, j] -= vAj
-            for i = 2:m
-                A[i, j] -= x[i]*vAj
-            end
-        end
+    @inbounds for j = 1:n
+        Aj, xj = view(A, 2:m, j), view(x, 2:m)
+        vAj = conj(τ)*(A[1, j] + dot(xj, Aj))
+        A[1, j] -= vAj
+        axpy!(-vAj, xj, Aj)
```

Comment:
I usually do `Aj .-= vAj .* xj` which I find more readable, but I'm not 100% sure performance is identical

Reply:
I compared both for BLASFloats and for BigFloats, and in both cases `axpy!` was not worse (BigFloats) or much better (Float64).